### PR TITLE
Fix BenchOpt issue 52

### PR DESCRIPTION
**Pull request by tanglef**
Use R for the whole PGD computation

### CODE REVIEW

**Comment by tanglef** [comment]
I just wanted to warn that the branch for the CI was modified  in the latest commit.
That was necessary to get the modification of PR #97 of BenchOpt which also passed the checks.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -25,13 +25,13 @@ jobs:
 
     - name: Install dependencies
       env:
-        BENCH_BRANCH: master
+        BENCH_BRANCH: Fix52
```
Comment (tomMoral) [inline .github/workflows/main.yml:28]:
```suggestion
        BENCH_BRANCH: master
```

**Review by tomMoral — APPROVED** [review]
LGTM.
Let's merge this one if the test pass with master from `benchopt` as the fix has been merged in benchopt/benchopt#97

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ jobs:
         conda activate base
         # make sure we have the latest version of pip to correctly install benchopt in sub conda env
         pip install --upgrade pip
-        pip install -U git+https://github.com/benchopt/benchopt@$BENCH_BRANCH
+        pip install -U git+https://github.com/tanglef/benchopt@$BENCH_BRANCH
```
Comment (tomMoral) [inline .github/workflows/main.yml:34]:
```suggestion
        pip install -U git+https://github.com/benchopt/benchopt@$BENCH_BRANCH
```

**Pull request merged**